### PR TITLE
Make checkpoints modifiable and modify them for the BCH/SV chains

### PR DESCRIPTION
**Pull request by ptschip**

### CODE REVIEW

File: src/unlimited.cpp [hunk 1]
```diff
@@ -544,6 +544,46 @@ void UnlimitedSetup(void)
...
     GenerateBitcoins(GetBoolArg("-gen", DEFAULT_GENERATE), GetArg("-genproclimit", DEFAULT_GENERATE_THREADS), Params());
+
+
+    // Modify checkpoints depending on whether BCH or SV fork
+    if (Params().NetworkIDString() == "main")
+    {
+        if (nMiningSvForkTime == 0)
+        {
+            // Nov 15th 2018 activate LTOR, DSV op_code
+            ModifiableParams().ModifiableCheckpoints().mapCheckpoints[556767] =
```
Comment (gandrewstone) [inline src/unlimited.cpp]: Nit: can we make a local reference for ModifiableParams().ModifiableCheckpoints() so it is not repeated so much (makes the source look nicer)

Comment (ptschip) [reply src/unlimited.cpp]: ok done

File: src/chainparams.cpp [hunk 2]
```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2010 Satoshi Nakamoto
+﻿// Copyright (c) 2010 Satoshi Nakamoto
```
Comment (sickpig) [inline src/chainparams.cpp:1]: A [BOM](https://en.wikipedia.org/wiki/Byte_order_mark) char has been added here.

**Review by sickpig — APPROVED** [review]
other than the BOM char nit I'm ok with the PR

**Pull request merged**